app/Http/Controllers/Auth/SocialiteController.php

```diff
@@ -166,6 +166,9 @@ private function setRolesFromClaim(string $provider, $user): bool
         ) {
             $roles = [];
             $attributes = $this->socialite_user->getRaw();
+            $idtoken = Arr::get($this->socialite_user->accessTokenResponseBody, 'id_token');
+            $jwtcontents = json_decode(base64_decode(explode(".", $idtoken)[1]));
```

Comment:
this assumes the variable is an array, and that it contains a . for every provider